spec/controllers/hq/partners_controller_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'rails_helper'
+require 'will_paginate/array'
```

Comment:
I'm not sure; this was outside the scope of my story; this is really a bugfix so the develop branch passes build, if you run a spec other than the entire test suite.  But yes; returning an array or activerecord association directly without calling `.paginate` on it first will pop an exception for `undefined method 'total_pages` for ...`